py/desitarget/gaiamatch.py

```diff
@@ -222,7 +237,8 @@ def sub_gaia_edr3(filename, objs=None):
 
     # ADM read the gaia sweep.
     cols = ["PARALLAX", "PARALLAX_IVAR",
-            "PMRA", "PMRA_IVAR", "PMDEC", "PMDEC_IVAR"]
+            "PMRA", "PMRA_IVAR", "PMDEC", "PMDEC_IVAR",
```

Comment:
It looks like you made changes on AEN in the MWS main survey cuts. Are you absolutely sure you don't want to use the EDR3 version of astrometric_excess_noise, which would require transferring it in here?

Reply:
Again, good catch. I've added AEN to this list and removed the GAIA prefix from APS.